simulators/ethereum/pyspec/README.md

```diff
@@ -0,0 +1,26 @@
+# Hive Pyspec
+
+This is a `simulator` for running the python [execution-spec-tests](https://github.com/ethereum/execution-spec-tests) within [`hive`](https://github.com/ethereum/hive), based on the `consensus` simulator. It differs mostly by using the `Engine API` to feed blocks into clients. 
+
+The simulator is only valid for post `Merge` forks.
+
+It can be run from the `hive` root directory simply, e.g for `geth`:
+```sh
+./hive --client go-ethereum --sim ethereum/pyspec
+```
+
+The `pyspec simulator` uses the latest test fixtures from the
+most recent execution-spec-tests [release](https://github.com/ethereum/execution-spec-tests/releases).
+
+
+### Running Specific Test Fixtures
+
+To run a specific or set of test fixtures `testCategory := ""`, within the third code section of `main.go/fixtureRunner()`, can be updated:
+- To only run the `large_withdrawals.json` test fixture: `testCategory := "withdrawals/withdrawals/large_withdrawals.json"`.
+- To run all the withdrawals test fixtures: `testCategory := "withdrawals"`.
```

Comment:
Not sure if you are aware, but the hive tool has an option for this: `hive --sim.limit <pattern>` will select tests based on a regular expression. The format of the pattern is `<suite>/<test>` and the suite/test components are matched individually. To run all suites, but only tests containing `foo` within them, run it hive like this `hive --sim.limit /foo`.

Reply:
I wasn't aware I could use this here! Thanks for the tip - I've addressed the latter the latest commit.